ertviz/models/plot_model.py

```diff
@@ -1,12 +1,16 @@
+import plotly.graph_objects as go
+
+
 class PlotModel:
-    def __init__(self, x_axis, y_axis, text, name, mode, line, marker):
+    def __init__(self, x_axis, y_axis, text, name, mode, line, marker, error_y=None):
```

Comment:
Consider using kwargs or making setters for this in the future. Too many arguments.

Reply:
Agree!